lib/helpers/campaign.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     }
 
     if (!campaign.endDate) {
-      return true;
+      return false;
```

Comment:
Now that the response for [Grab the Mic](https://next.dosomething.org/api/v2/campaigns/8017) is looking better, I realized if the `endDate` is null, status should be parsed as `'active'`.

(although.. the `headline` in all caps may come across as quite shouty if we don't override the campaign `askSignup` template)